ClassLibrary/ClassLibrary.Test/UnitTest1.cs

```diff
@@ -5,18 +5,31 @@ namespace ClassLibrary.Test
     [TestFixture]
     public class UnitTest1
     {
+        Library lib = new Library();
+
         [Test]
         public void TestBooksExist()
         {
-            Library l = new Library();
-            Assert.AreEqual(true, l.Books!=null);
+            
+            Assert.AreEqual(true, lib.Books!=null);
         }
 
         [Test]
         public void TestBooks()
         {
-            Library l = new Library();
-            Assert.AreEqual(true, l.Books?.Count>=0);
+            Assert.AreEqual(true, lib.Books?.Count==0);
+        }
+
+        [Test]
+        public void TestAuthorExist()
+        {
+            Assert.AreEqual(true, lib.Authors != null);
+        }
+
+        [Test]
+        public void TestAuthors()
```

Comment:
It would help if you named test method according to something we test. Also it's a good practice to name test methods like this: "WhenX_ShouldY". In this case, `WhenLibraryCreated_ShouldHaveNoAuthors`